doc/api/modules.md

```diff
@@ -171,18 +171,19 @@ relative, and based on the real path of the files making the calls to
 ## Loading ECMAScript modules using `require()`
 
 <!-- YAML
-added: v22.0.0
+added:
+  - v22.0.0
+  - v20.17.0
```

Comment:
This change looks unrelated, also on release branches we only keep track of the changes for that release line.
```suggestion
added: v22.0.0
```